_wcs/boards/GiftCard/gift-card.board.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { createBoard } from '@wixc3/react-board';
 import { GiftCard } from '../../../src/components/gift-card/gift-card';
-import photo from '../../../src/components/gift-card/assets/img-2.jpg';
+import photoURL from '../../../src/components/gift-card/assets/img-2.jpg';
 import styles from './gift-card.board.module.scss';
 
 export default createBoard({
     name: 'Example of a gift card panel',
-    Board: () => <div className={styles.grid2col}><GiftCard imgSrc={photo} /><GiftCard /></div>,
+    Board: () => <div className={styles.grid2col}><GiftCard photoURL={photoURL} /><GiftCard /></div>,
```

Comment:
Its good practice to first refctor the component, and then make the boards.

or vice versa